lib/flowViewport/sceneIndex/fvpDefaultMaterialSceneIndex.cpp

```diff
@@ -22,13 +22,33 @@
 #include <pxr/imaging/hd/materialBindingsSchema.h>
 #include <pxr/imaging/hd/containerDataSourceEditor.h>
 #include <pxr/imaging/hd/retainedDataSource.h>
+#include <pxr/usd/usdGeom/tokens.h>
 
 namespace FVP_NS_DEF {
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace{
     static const TfToken purposes[] = { HdMaterialBindingsSchemaTokens->allPurpose };
+
+    // Support implicit surfaces from USD as well, not only meshes
+    bool _IsDefaultMaterialCompliantPrimitive(const TfToken& primType) 
+    { 
+        static VtArray<TfToken> const compliantPrimitives = { HdPrimTypeTokens->cone,
```

Comment:
Minor: would be slightly better as an std::set, with the check using either std::set::find() or std::set::count().